src/main/scala/diplomacy/LazyModule.scala

```diff
@@ -136,14 +120,15 @@ object LazyModule
   protected[diplomacy] var scope: Option[LazyModule] = None
   private var index = 0
 
-  def apply[T <: LazyModule](bc: T)(implicit sourceInfo: SourceInfo): T = {
+  def apply[T <: LazyModule](bc: T)(implicit valName: ValName, sourceInfo: SourceInfo): T = {
```

Comment:
Consider introducing this API change now to keep things aligned while we await a solution for Chisel module disambiguation

Reply:
https://github.com/freechipsproject/rocket-chip/pull/1159